lib/ruby2d/text.rb

```diff
@@ -11,6 +11,10 @@ def initialize(x=0, y=0, text="Hello World!", size=20, font=nil, c='white', z=0)
       
       # if File.exists? font
         @font = font
+      end
+      
+      # if File.exists? font
+        # @font = font
       # else
```

Comment:
Oh, no, sorry, the code is ok, but I didn't remove earlier commented out code. The plan is that it is acceptable to provide nil as font, and code will find a font for the user. You don't do any Font.new, simply the expectation is that Font.default should return an path to file with working font. Maybe #default_font_path would be better name?